src/mcts/search.cc

```diff
@@ -192,11 +192,14 @@ int64_t Search::GetTimeToDeadline() const {
 }
 
 namespace {
-inline float GetFpu(const SearchParams& params, Node* node, bool is_root_node) {
+inline float GetFpu(const SearchParams& params, Node* node,
+                    bool is_root_node, bool logit_q = false) {
   const auto value = params.GetFpuValue(is_root_node);
+  const auto scale = params.GetOneMinusEps();
+  const auto Q = logit_q ? FastLogit(scale * node->GetQ()) : node->GetQ();
   return params.GetFpuAbsolute(is_root_node)
-             ? value
-             : -node->GetQ() - value * std::sqrt(node->GetVisitedPolicy());
+             ? (logit_q ? FastLogit(scale * value) : value)
```

Comment:
I would suggest to replace ``(logit_q ? FastLogit(scale * value) : value)`` by ``(logit_q ? FastLogit(scale * std::max(-1.0f,value)) : value)`` to avoid unwanted behavior since we don't guarantee that ``FpuValue`` is in ``[-1,1]``. Having that value at -1.0 still basically guarantees that that node will never be explored, so one should refrain from using absolute fpu -1.0 with logit.

Reply:
Note that -1 is the value we usually use with absolute.

Reply:
I did `min(1, max(-1, value))` to bound on both sides.